### ISSUE TITLE
NTR: DHCA (dihydrocholestanoic acid) high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8254]
New term request DHCA (dihydrocholestanoic acid) high in blood (segal_070920164116):   UMLS: 4229201

**Comment by pnrobinson** [comment]
@MickeySegal
NOTE CORRECTION OF SPELLING

Adding new term: 
- ID: HP:6000831
- label: Increased circulating dihydroxycholestanoic acid concentration
- parents: [Increased serum bile acid concentration (HP:0012202)](https://hpo.jax.org/app/browse/term/HP:0012202)
- pmid: PMID:26453805
- def: The concentration of dihydroxycholestanoic acid (DHCA) in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8253
- synonyms:

**Comment by MickeySegal** [comment]
Some NTRs are discussed in more than one report.  This one we discussed in https://github.com/obophenotype/human-phenotype-ontology/issues/8837 but no HPO code was assigned until now.
